app/src/main/java/org/stepik/android/domain/course_list/interactor/CourseListInteractor.kt

```diff
@@ -62,7 +62,7 @@ constructor(
         sourceTypeComposition: SourceTypeComposition
     ): Single<PagedList<CourseListItem.Data>> =
         courseStatsInteractor
-            .getCourseStats(courses, resolveEnrollmentState = false, sourceTypeComposition = sourceTypeComposition)
+            .getCourseStats(courses, resolveEnrollmentState = true, sourceTypeComposition = sourceTypeComposition)
```

Comment:
`resolveEnrollmentState` always `true` now, in past that causes very slow loading of paid courses